src/decider/agent/copyrightDeactivationClutterRemovalScript.py

```diff
@@ -0,0 +1,187 @@
+    for index, row in df.iterrows():
+        text = df.loc[index, 'content']
+        doc = nlp(text)
+
+        if type(text) == float:
+            continue
+        
+        ## Lemmatization
+        lemma_list = []
+        for token in doc:
+            lemma_list.append(token.lemma_)
+        
+        # Filter the stopword
+        filtered_sentence =[] 
+        for word in lemma_list:
+            lexeme = nlp.vocab[word]
+            if lexeme.is_stop == False:
+                filtered_sentence.append(word)
+        
+        # Remove punctuation
+        punctuations="?:!.,;'"
+        for word in filtered_sentence:
+            if word in punctuations:
+                filtered_sentence.remove(word)
+
+        ## List joining and Filtering (c) and copyright unicode symbol
+        list_of_copyrights = " ".join(map(str,filtered_sentence))
+        substring = "( c )"
+        cp_symbol = '\xa9' ##Unicode for copyright Symbol
+        
+        if "copyright" not in list_of_copyrights:
+            if substring in list_of_copyrights:
+                list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+            
+            elif cp_symbol in list_of_copyrights:
+                list_of_copyrights = list_of_copyrights.replace(cp_symbol, "copyright")
+                
+        if substring in list_of_copyrights:
+            list_of_copyrights = list_of_copyrights.replace(substring, "copyright")
+
+        elif cp_symbol in list_of_copyrights:
+            list_of_copyrights = list_of_copyrights.replace(cp_symbol, "copyright")
+    
+        ## Implementing NER and POS Tags after normalization
+        doc2 = nlp(list_of_copyrights)
+
+        ## All the NER taggings will be contained in a dictionary having "Entity" and "Values" as keys
+        ent_dict = {}
+
+        full_table_ner = { "Entity": [], "Values": []}
+
+        for x in doc2.ents:
+            ent_dict[x.text] = x.label_
+
+        for key in ent_dict:
+            full_table_ner["Entity"].append(key)
+            full_table_ner["Values"].append(ent_dict[key])
+        
+        ## All the POS taggings will be contained in a dictionary having "Entity" and "POS_TAGS" as keys
+        pos_dict = {}
+        full_table_pos = { "Entity": [], "POS_TAG": []}
+
+        for token in doc:
+            if not token.is_punct | token.is_space:
+                pos_dict[token.text] = token.pos_
+
+        for key in pos_dict:
+            full_table_pos["Entity"].append(key)
+            full_table_pos["POS_TAG"].append(pos_dict[key])
+
+        ## The checking function call happening with each iteration
+        entityCheck(full_table_pos, full_table_ner, index, clutter_flag, df)
+    print(df.to_json(orient='records'))
+    
+
+def entityCheck(listA, listB, index, clutter_flag, df):
+    if set(THE_PROBABLE_LOGIC_POS_CHECK).intersection(set(listA["POS_TAG"])):
+        if set(THE_PROBABLE_LOGIC_NER_CHECK).intersection(set(listB["Values"])):
```

Comment:
```suggestion
    if set(THE_PROBABLE_LOGIC_POS_CHECK).intersection(set(listA["POS_TAG"])) and set(THE_PROBABLE_LOGIC_NER_CHECK).intersection(set(listB["Values"])):
```
Can be combined together for better readability.

Reply:
Nested if..else is there. Can't be combined.

Reply:
Thanks for pointing it out.